src/docfx/Models/WatchCommandOptions.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using Spectre.Console.Cli;
+
+namespace Docfx;
+
+[Description("Generate client-only website combining API in YAML files and conceptual files and watch them for changes")]
+internal class WatchCommandOptions : DefaultBuildCommandOptions
+{
+    [Description("Should directory be watched and website re-rendered on changes.")]
+    [CommandOption("-w|--watch")]
+    [DefaultValue("true")]
+    public bool Watch { get; set; }
```

Comment:
The `--watch` option looks unnecessary, when would we want to execute `docfx watch` without watching?